### PR TITLE
Allow passing array payload and criteria

### PR DESCRIPTION
**Pull request by fschmtt**
* Allow passing `array` as payload to `Command`
* Allow passing `Criteria` to a `Command`

### CODE REVIEW

File: src/Resource/Users.php [hunk 1]
```diff
@@ -204,16 +204,25 @@ public function removeRealmRoles(string $realm, string $userId, RoleCollection $
...
-    public function executeActionsEmail(string $realm, string $userId): void
+    /**
+     * @param list<string>|null $actions
+     */
+    public function executeActionsEmail(string $realm, string $userId, ?array $actions = null, ?Criteria $criteria = null): void
     {
+        $payload = $actions ? [
+            'actions' => $actions,
+        ] : null;
+
```
Comment (packagespace) [inline src/Resource/Users.php:215]: Hi, I was trying out the fork of this branch and it seems like there's a mistake here. The payload needs to be only the array itself (e.g. `["UPDATE_PASSWORD"]`). How should I go about contributing if I'd like to fix this and help get this merged?

Comment (fschmtt) [reply src/Resource/Users.php]: Hey, thanks for trying it out 🙂 

I changed it to
```php
    /**
     * @param list<string>|null $actions
     */
    public function executeActionsEmail(string $realm, string $userId, ?array $actions = null, ?Criteria $criteria = null): void
    {
        $this->commandExecutor->executeCommand(
            new Command(
                '/admin/realms/{realm}/users/{userId}/execute-actions-email',
                Method::PUT,
                [
                    'realm' => $realm,
                    'userId' => $userId,
                ]
                ],
                $actions,
                $criteria,
            )
        );
    }
```

Can you verify that this works now as intended?

Comment (packagespace) [reply src/Resource/Users.php]: Yup, this works!

Comment (fschmtt) [reply src/Resource/Users.php]: Nice, thanks a lot for your help!

**Pull request merged**